inspirehep/dojson/hep/fields/bd90x99x.py

```diff
@@ -54,6 +57,11 @@ def get_value(value):
                 year = int(value.get('y'))
             except:
                 pass
+        if 'i' in value:
+            try:
+                isbn = normalize_isbn(value.get('i'))
+            except:
```

Comment:
Do you know what kind of error/exception it may throw? If so could you insert it after `except`?